rip/generic_steps/default_schema_validation.py

```diff
@@ -27,6 +27,10 @@ def _get_fields_to_validate_data(self, request, data):
     def validate_data(self, request, data):
         errors = {}
         fields_to_validate = self._get_fields_to_validate_data(request, data)
+
+        if data is not None and type(data) != dict:
+            return "This item should be an object."
```

Comment:
This field should be an object is a better message, than dict (which is python specific)